lib/note-content-editor.tsx

```diff
@@ -515,6 +601,38 @@ class NoteContentEditor extends Component<Props> {
       window.electron ? false : true
     );
 
+    editor.addCommand(
+      monaco.KeyMod.WinCtrl | monaco.KeyCode.Space,
+      function () {
+        // only show suggestions if we have an open bracket
+        const model = editor.getModel();
+        if (!model) {
+          return;
+        }
+        // @todo DRY - code duplicated from the completion handler
+        const position = editor.getPosition();
+        const line = model.getLineContent(position.lineNumber);
+        if (!line.includes('[')) {
+          return;
+        }
+        const precedingOpener = line.lastIndexOf('[', position.column);
+        const precedingCloser = line.lastIndexOf(']', position.column);
+        const precedingBracket =
+          precedingOpener >= 0 && precedingCloser < precedingOpener
+            ? precedingOpener
+            : -1;
+
+        const soFar =
+          precedingBracket >= 0
+            ? line.slice(precedingBracket + 1, position.column)
+            : '';
+
+        if (soFar) {
+          editor.trigger('suggestions', 'editor.action.triggerSuggest', null);
+        }
+      }
+    );
```

Comment:
nitpick: even though we don't use `this` here I do like using function shorthand syntax since it ensures that if we run into `this` issues we will know that `this` is lexically-scoped and not dynamically-scoped as is done when using `function(){}` syntax

not a big deal - just noted the syntax